commands/deploy.go

```diff
@@ -57,22 +67,29 @@ func (c *DeployCmd) Run() (err error) {
 		return fmt.Errorf("failed to upload tarball: file size (%d) is greater than (%d)", stat.Size(), MaxFileSize)
 	}
 
-	fmt.Println("Deploying...")
+	fmt.Println("Pushing...")
 
 	client := &http.Client{
 		Timeout: 30 * time.Second,
 	}
-	req, err := http.NewRequest(http.MethodPost, c.ServerURL, tempFile)
+	req, err := http.NewRequest(http.MethodPost, c.ServerURL.String(), tempFile)
 	if err != nil {
 		return fmt.Errorf("failed to create upload URL: %v", err)
 	}
+
+	username, password, err := auth.GetCredential(c.ServerURL.Host)
```

Comment:
Where is the password stored?

Reply:
This was introduced in #7. 

Password is [pulled from](https://github.com/section/sectionctl/blob/0d157b5fb78eb2ddc75769aeb1a8568cae3f70fe/api/auth/auth.go#L135-L147) a netrc in `~/.config/section`.